ASUWebPlatforms/webspark-ci
# Ws2 1602: asu_degree_rfi: Update services used in module UI to transition from Degree Search REST API to Data Potluck Academic Plan codeset

### Description

To test:
- Go to `/degree-listing-page` and click on one of the degrees to generate a new degree detail page
- If the page generates, this is good.
- Log in with admin permissions
- Go to `/bachelors-degrees/majorinfo/ASACOCBS/undergrad/false/37`
- Click on "Layout" and scroll down to the RFI form and click the pencil icon, and then click "Configure."
- If the form loads with data in the following fields, this should pass:
  - College
  - Department code
  - Campus
  - Area of interest
  - Program of interest
  - Country
  - State or province

### Links

- [JIRA ticket](https://asudev.jira.com/browse/WS2-1602)

### Checklist
